applications/brain_dataset_tutorial.md

```diff
@@ -0,0 +1,77 @@
+# Using Dask and Napari to view and process 100µm resolution human brain dataset.
+We used napari to view and process 100µm resolution human brain large dataset, and in the following tutorial we will break down how it all builds up!
+
+The dataset was obtained from Dr. Brian Edlow's Lab for NeuroImaging of Coma and Consciousness, which is dedicated to promoting recovery of consciousness in people with severe brain injuries.
+This dataset is a 100 micron resolution magnetic resonance imaging (MRI) scan of an ex vivo human brain specimen. The brain specimen was donated by a 58-year-old woman who had no history of neurological disease and died of non-neurological causes.
+
+A very important concept to bear in mind when going through this tutorial is the [Lazy Loading](https://en.wikipedia.org/wiki/Lazy_evaluation), i.e., only loading images to memory when the position slider on napari maps to that particular image, which is very of use, when using large datasets.
+
+#### So how can we lazily load an image using dask?
+First, we import delayed from dask, which is the main function responsible for handling __Lazy Loading__. Here we defined a function *load_image()* that will lazy load the image when called given, the image name as a parameter. 
+```
+
+from dask import delayed
+import dask.array as da
+def load_image(image_name):
+	delayed_image_load = delayed(io.imread)
+	return da.from_delayed(delayed_image_load(image_name), dtype='float64', shape=(1760, 1760))
+```
+#### Getting the data
+
+You can download the dataset from [here](https://datadryad.org/stash/downloads/file_stream/223913). A ZIP file will be downloaded, after unzipping the compressed dataset, the `.tiff` images will be extracted to the `SYNTHESIZED_TIFF_Images_Raw/Synthesized_FLASH25_100um_TIFF_Axial_Images/` directory. Now your dataset is ready to be used!
```

Comment:
```suggestion
You can directly download the dataset from [here](https://datadryad.org/stash/downloads/file_stream/223913). A ZIP file will be downloaded, after unzipping the compressed dataset, the `.tiff` images will be extracted to the `SYNTHESIZED_TIFF_Images_Raw/Synthesized_FLASH25_100um_TIFF_Axial_Images/` directory. Now your dataset is ready to be used!
```
Can you add a comment about how big the dataset is? It can be a little unnerving to have a file start downloading directly when clicking on a link

Reply:
Yes, that is pretty unnerving. Could you link to the main page of this dataset on Dryad?

It could go something like this:
```
You can download the dataset from [here](url/of/main/dyrad/dataset/page) (or use the [direct download link](https://datadryad.org/stash/downloads/file_stream/223913), dataset is ???GB). A ZIP file will be downloaded...
```

Reply:
I found the link to the main dataset page! https://doi.org/10.5061/dryad.119f80q